css/main.css

```diff
@@ -198,25 +235,93 @@ a.logo{
 .breadcrumb > .active {
   color: #f03c3c;
 }
-
+.container-blog-gridzilla{
+    padding-left: 18px;
+    padding-right: 20px;
+}
+.col-blog-gridzilla{
+    padding-left: 10px;
+    padding-right: 10px;    
+}
 /* Article_post*/
 .post{
-    padding-bottom: 0 10px 10px 10px;
+    padding-bottom: 21px;
 }
 .post-imgWrapper img{
     width: 100%;
 }
-.post-hover:hover{
-    background-color: rgba(0, 0, 0, 0.75);
+.post-imgWrapper{
+    position: relative;
+}
+.post-imgWrapper:hover .post-hover{
+    display: list-item;
+}
+.post-hover{
+    position: absolute;
+    top: 0;
+    left: 0;
     height: 100%;
+    width: 100%;
+    color: #fff;
+    background-color: rgba(0, 0, 0, 0.75);
+    display: none;
+}
+.post-hover img{
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    margin: auto;
+    max-width: 37px;
+    max-height: 37px;
+}
+.post-hover a{
+    color: inherit;
+}
+.post-hover-buttons{
+    position: absolute;
+    bottom: 20px;
+    width: 100%;
+}
+.imgVisit, 
+.imgDate, 
+.imgLike {
+    display: block;
+    float: left;
+    width: 33%;
+    text-align: center;
+}
+.imgVisit:before, 
+.imgDate:before,
+.imgLike:before {
+    content: '';
+    background: no-repeat url('../img/visit.png');
+    display: block;
+    height: 13px;
+    width: 24px;
+    margin: 0 auto;
+    padding-bottom: 20px
+}
+
+.imgDate:before {
+    background: no-repeat url('../img/clock.png');
+    height: 14px;
+    width: 15px;
+}
+ 
+.imgLike:before {
+    background: no-repeat url('../img/like.png');
```

Comment:
can you create a sprite for `../img/visit.png`, `../img/clock.png`, `../img/like.png`?